scripts/test/__init__.py

```diff
@@ -38,7 +39,7 @@ def test_dist(clean_cache: bool = False):
     steps.extend([
```

Comment:
These steps are a subset of what is in `test_all`, so can you create a function for adding these steps that avoids the repeated code in `test_all()` and `test_dist()` ?

Reply:
Done